src/libcore/convert.rs

```diff
@@ -213,7 +213,7 @@ pub trait AsMut<T: ?Sized> {
 /// # Generic Implementations
 ///
 /// - [`From`]`<T> for U` implies `Into<U> for T`
-/// - `Into` is reflexive, which means that `Into<T> for T` is implemented
+/// - [`Into`] is reflexive, which means that `Into<T> for T` is implemented
```

Comment:
There are no links to [`Into`]. You could write one.

Also the link in line 276 is unused:
https://github.com/rust-lang/rust/blob/a6a8f6c5b37597bf69178ee5cc6c185b7137f5d8/src/libcore/convert.rs#L276-L282

Reply:
Thanks. PTAL